charts/tekton-apps/templates/general/triggerbindings.yaml

```diff
@@ -39,13 +39,26 @@ spec:
     value: {{ $.Values.defaultRegistry }}
   {{- end }}
   {{- toYaml $component.triggerBinding | nindent 2 }}
+
+  {{- if $argocdSource }}
+  # ! OLD WAY:
+  # If you see this message - pls transfer kubernetes manifests into dedicated kubernetes git repo
+  # if we want to support old ways when our kubernetes manifests where specifically declared inside
+  # actual backend or frontend repos. In new approach we have isolated kubernetes repository with manifests
   - name: kubernetes_repository_ssh_url
     value: {{ $argocdSource.repoUrl | default ($project.kubernetesRepository).url }}
```

Comment:
Let's remove default values related to `kubernetesRepository` in this section then?

Reply:
I think it's a good safety to have. Plus is there an option that argocdSource.repoUrl is not defined? Or is it ALWAYS going to be defined?